setup.py

```diff
@@ -138,6 +138,9 @@ def finalize_options(self):
         'pybind11>=1.7',
         'psutil',
         'nibabel',
+        'typing',
```

Comment:
Are we using features from `typing` that are not in Python 3.6? If not, the version in the standard library will work.

Reply:
I haven't put any type annotations in so I don't know how this made it in there (and actually, I think it may have been breaking some of the travis builds?). Fixed.